packages/crypto/src/lib/crypto.ts

```diff
@@ -191,17 +191,50 @@ export const combineBlsShares = (
  *
  */
 export const combineEcdsaShares = (sigShares: Array<SigShare>): any => {
-  // R_x & R_y values can come from any node (they will be different per node), and will generate a valid signature
-  const R_x = sigShares[0].localX;
-  const R_y = sigShares[0].localY;
+
+  console.log("sigShares:", sigShares);
 
   // the public key can come from any node - it obviously will be identical from each node
   // const publicKey = sigShares[0].publicKey;
   // const dataSigned = '0x' + sigShares[0].dataSigned;
-  const validShares = sigShares.map((s: any) => s.shareHex);
-  const shares = JSON.stringify(validShares);
+  // filter out empty shares
+  const validShares = sigShares.reduce((acc, val) => {
+    if (val.shareHex !== '') {
+      acc.push(val.shareHex);
```

Comment:
this should be 
```suggestion
      acc.push(val);
```

otherwise it is breaking right now.

Also doesn't seem like we have any active cypress tests that test this / the `litNodeClient.executeJs` call?